sftp/src/main/scala/blobstore/sftp/SftpPath.scala

```diff
@@ -0,0 +1,17 @@
+package blobstore.sftp
+
+import java.time.Instant
+
+import blobstore.Path
+import cats.data.Chain
+import com.jcraft.jsch.SftpATTRS
+
+case class SftpPath(root: Option[String], fileName: Option[String], pathFromRoot: Chain[String], attributes: SftpATTRS)
```

Comment:
Nice! Wasn't sure if `SftpATTRS` brings anything useful except 3 already exposed attributes.

Reply:
I think `uid`/`gid` could be useful for filtering, doesn't hurt to expose creation date either. We could maybe map all these fields, not just a subset of them. I'll do it in another PR.